js/flightlog_fields_presenter.js

```diff
@@ -593,12 +593,26 @@ function FlightLogFieldPresenter() {
                 case 'STACK':
                     return value.toFixed(0);
                 case 'FFT':
-                    return Math.round(flightLog.gyroRawToDegreesPerSecond(value)) + "deg/s";
+                    switch (fieldName) {
+                    case 'debug[0]': // gyro scaled [for selected axis]
+                    case 'debug[1]': // pre-dyn notch gyro [for selected axis]
+                    case 'debug[2]': // pre-dyn notch gyro FFT downsampled [roll]
+                        return Math.round(flightLog.gyroRawToDegreesPerSecond(value)) + "deg/s";
+                    case 'debug[3]': // FFT bin mean index
+                        return (value / 100).toFixed(2);
+                    }
+                    break;
                 case 'FFT_TIME':
-                    return value.toFixed(0) + "\u03BCS";
+                    switch (fieldName) {
+                    case 'debug[1]':
+                    case 'debug[2]':
+                        return value.toFixed(0) + "\u03BCs";
+                    }
```

Comment:
What happens with debug[0]?  I understand that it goes to the default, that returns the value directly, but without "units". I don't know if the 'Active calc step' can have a unit for it.

Reply:
The step is really unitless:
```c
DEBUG_SET(DEBUG_FFT_TIME, 0, state->updateStep);
```
where `updateStep` is:
```c
enum {
        STEP_ARM_CFFT_F32,
        STEP_BITREVERSAL,
        ...
        STEP_HANNING,
        STEP_COUNT
    };
```

Reply:
No, we have not for this. We can add it but I don't think the effort worth it for this. So to me the code is ok.

Reply:
Only for curiosity, we have the enumerations function in other values: https://github.com/betaflight/blackbox-log-viewer/blob/77e1ba3524b405dcdb6f365215a227d183cdc2ea/js/flightlog_fields_presenter.js#L498-L505

Reply:
Thanks, that's helpful.
If you think it will be useful I can add the FFT step enum?

btw my test fft logs always show only step 1 in the trace. I wonder if it has something to do with the logging rate (it was 1kHz).

Reply:
As you prefer. If you add it, it will be totally finish. If not, it works as is, so no problem.